src/pages/index.astro

```diff
@@ -1,9 +1,10 @@
 ---
 import BaseLayout from '../layouts/BaseLayout.astro';
-
+import Footer from '../components/Footer.astro';
 import AboutUs from '../components/AboutUs.astro';
 ---
 
 <BaseLayout>
   <AboutUs />
 </BaseLayout>
+<Footer />
```

Comment:
Remove it this component from this page and place it inside the `BaseLayout` Component